examples/authentication/pom.xml

```diff
@@ -11,7 +11,8 @@
     <artifactId>authentication</artifactId>
     <packaging>war</packaging>
 
-    <name>WildFly bootable jar authentication Example</name>
```

Comment:
The convention we try to apply is WildFly bootable JAR.

Reply:
Thanks, I updated the names